telegraf/opt/intel-gpu-telegraf/get_intel_gpu_status.sh

```diff
@@ -5,9 +5,25 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-JSON=$(/usr/bin/timeout -k 3 3 /usr/bin/intel_gpu_top -J)
-VIDEO_UTIL=$(echo "$JSON"|grep Render/3D -A 1|tail -1|grep busy|cut -d ":" -f2|cut -d "," -f1|cut -d " " -f2)
+CARDS=$(/usr/local/bin/intel_gpu_top -L | grep card1)
```

Comment:
I believe this will fail on a non arc system because card1 won't exist. it might be good to have card1 as a variable in case we need to modify it in the future.

Reply:
Yes, this PR only supports iGPU and dGPU arc devices. 
We can support flex cards in the future